vts/trustedservices/trustedservices_grpc.go

```diff
@@ -360,38 +360,41 @@ func (o *GRPC) GetAttestation(
 		return o.finalize(appraisal, err)
 	}
 
-	appraisal.EvidenceContext.ReferenceId = extracted.ReferenceID
+	appraisal.EvidenceContext.ReferenceIds = extracted.ReferenceIDs
 
 	o.logger.Debugw("constructed evidence context",
-		"software-id", appraisal.EvidenceContext.ReferenceId,
-		"trust-anchor-id", appraisal.EvidenceContext.TrustAnchorId)
+		"software-id", appraisal.EvidenceContext.ReferenceIds,
+		"trust-anchor-id", appraisal.EvidenceContext.TrustAnchorIds)
 
-	endorsements, err := o.EnStore.Get(appraisal.EvidenceContext.ReferenceId)
-	if err != nil && !errors.Is(err, kvstore.ErrKeyNotFound) {
-		return o.finalize(appraisal, err)
-	}
+	var multEndorsements []string
+	for _, refvalID := range appraisal.EvidenceContext.ReferenceIds {
```

Comment:
Note for the future (not necessary for this pull): Rather that looping over multiple requests to the store, we should add a Store API to handle multiple IDs  in a single request, e.g. so that his can just be `o.EnStore.GetMultiple(appraisal.EvidenceContext.ReferenceId)`.

Reply:
@setrofim Valid point, I will raise an issue for the same!

Reply:
https://github.com/veraison/services/issues/207 raised to track this enhancement!